updater/wyc.go

```diff
@@ -99,16 +99,24 @@ type wycConfig struct {
 	PublicKey      string
 }
 
-// GetWYSURLs returns the ServerFileSite(s) listed in the WYC file.
+// GetWYSURLs returns the ServerFileSite(s) listed in the WYC file associated with config and populates
+// urls with the site URLs.
+// args are used to inject any CLI provided URL arguments and allow for overriding of the site URL.
 func (config ConfigIUC) GetWYSURLs(args Args) (urls []string) {
 	// WYS URL specified on the command line
-	if len(args.Server) > 0 {
-		urls = append(urls, args.Server)
-		return urls
+	urlsToConsider := []string{}
+	for _, fileSite := range config.IucServerFileSite {
+		urlsToConsider = append(urlsToConsider, string(fileSite.Value))
```

Comment:
```suggestion
	urlsToConsider := make([]string, len(config.IucServerFileSite))
	for idx, fileSite := range config.IucServerFileSite {
		urlsToConsider[idx] = string(fileSite.Value)
```
Is technically more performant, but it doesn't matter much here.

Reply:
👍